src/Core/Telemetry/ITelemetryManager.cs

```diff
@@ -28,5 +28,7 @@ public interface ITelemetryManager
         void Update();
         void LanguageAnalyzed(string languageKey);
         void ShowHotspotRequested();
+        void TaintIssueOpened();
+        void TaintIssueOpenedInBrowser();
```

Comment:
I think calling the methods something like `TaintIssueOpenedLocally/OnClient` and `TaintIssueOpenedOnServer` would be slightly clearer, but it doesn't matter too much.